Python Part/lib/fnn.py

```diff
@@ -0,0 +1,76 @@
+import numpy as np
+from lib.sigmoid import sigmoid
+from utils.report_generator import generate_report
+
+
+class fnn_layers:
+    def __init__(self, input_neurons, output_neurons):
+        self.input_neurons = input_neurons
+        self.output_neurons = output_neurons
+
+        np.random.seed(1000)
+
+        print("\n\n")
+        print("built a neural network with %d input, %d output" % (
+            self.input_neurons, self.output_neurons))
+        print("\n\n")
+        self.__output_weights = np.random.uniform(-1, 1, size=(
+            self.output_neurons, input_neurons))
+
+    def evaluate_model(self, epoch_size, alpha, images_train, labels_train, images_test, labels_test, model_name):
```

Comment:
this function is repeated, find a way to generalize it and move it to another file.